src/raygun.rum.js

```diff
@@ -193,6 +200,16 @@ var raygunRumFactory = function(window, $, Raygun) {
       }
     };
 
+    this.trackCustomTiming = function(name, duration, offset) {
+      if(typeof duration === "number") {
+        var newTimings = [];
+        newTimings.push(createCustomTimingMeasurement(name, duration, offset));
+        addPerformanceTimingsToQueue(newTimings, false);
+      } else {
+        log('Raygun4JS: Custom timing "' + name + '" duration is value is not a number');
```

Comment:
Can remove the 'is' here. So it would be:

" duration value is not a number"

Reply:
Fixed in a seperate PR